.github/workflows/golangci-lint.yaml

```diff
@@ -0,0 +1,14 @@
+name: golangci-lint
```

Comment:
is there any specific reason to break this out into its own workflow, rather than adding it as a job to the existing prbuild.yaml workflow file (in place of the existing `lint` job)? If no specific reason, my pref would be to keep it in one workflow file for simplicity.

Reply:
First I thought this should run on push to master too. Then decided to keep it for PRs, And yes I move it to `prbuild.yaml`